device-types/Cisco/Meraki-MS225-24.yaml

```diff
@@ -0,0 +1,81 @@
+---
+manufacturer: Cisco
+model: Meraki MS225-24
+slug: cisco-meraki-ms225-24
+part_number: MS225-24-HW
+u_height: 1
+is_full_depth: false
+weight: 6.03
+weight_unit: lb
```

Comment:
2. Please add the `airflow` data point if you have it or can find it. This appears to be in the datasheet (passive)